nemo_aligner/data/nlp/datasets.py

```diff
@@ -403,10 +432,153 @@ def __getitem__(self, idx):
             "rejected_labels": labels_reject_tokens,
             "chosen_reward": payload.get("chosen_reward", self.default_chosen_reward),
             "rejected_reward": payload.get("rejected_reward", self.default_rejected_reward),
+            "ignore_example": ignore_example,
         }
         return output
 
 
+class DPOPackedDataset(DPOModelDataset):
```

Comment:
Can you write a unit test for this? There are some examples in the tests/test_dataset.py file

Reply:
Yes, will do